.travis.yml

```diff
@@ -30,6 +30,8 @@ env:
 install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - git clone https://github.com/Eficent/knowledge.git -b 12.0-mig-knowledge ${HOME}/knowledge_repo
+  - git mv ${HOME}/knowledge_repo/knowledge ${HOME}
```

Comment:
I think it should be `${HOME}/knowledge`

Reply:
thanks

Reply:
not working

Reply:
Are you sure the repo is downloaded there? You can try to do it after `travis_install_nightly`

Reply:
still not working